app/src/app/pages/plando-page/plando-items/plando-items.component.ts

```diff
@@ -0,0 +1,1218 @@
+    "Letter (To Merlow)",
+    "Letter (To Minh T.)",
+    "Letter (To Miss T.)",
+    "Letter (To Mort T.)",
+    "Letter (To Mr. E.)",
+    "Letter (To Muss T.)",
+    "Letter (To Nomadimouse)",
+    "Letter (To Red Yoshi Kid)",
+    "Letter (To Russ T.)",
+    "LifeShroom",
+    "Lime",
+    "LimeCandy",
+    "LuckyDay",
+    "LunarStone",
+    "Lyrics",
+    "MagicalBean",
+    "MagicalSeed1",
+    "MagicalSeed2",
+    "MagicalSeed3",
+    "MagicalSeed4",
+    "Mailbag",
+    "MapleShroom",
+    "MapleSuper",
+    "MapleSyrup",
+    "MapleUltra",
+    "MegaHPDrain",
+    "MegaJump",
+    "MegaQuake",
+    "MegaRush",
+    "MegaSmash",
+    "Melody",
+    "Melon",
+    "MiniJumpCharge",
+    "MiniSmashCharge",
+    "MiracleWater",
+    "Mistake",
+    "MoneyMoney",
+    "Multibounce",
+    "Mushroom",
+    "Mystery",
+    "MysteryNote",
+    "NuttyCake",
+    "OddKey",
+    "PDownDUp",
+    "POWBlock",
+    "PUpDDown",
+    "Parakarry",
+    "ParakarryUpgrade",
+    "PayOff",
+    "Pebble",
+    "Peekaboo",
+    "PotatoSalad",
+    "PowerBounce",
+    "PowerJump",
+    "PowerPlus",
+    "PowerQuake",
+    "PowerRush",
+    "PowerSmash",
+    "PowerStar",
+    "PrettyLucky",
+    "PrisonKey",
+    "ProgressiveBoots",
+    "ProgressiveHammer",
+    "ProgressiveJumpCharge",
+    "ProgressivePowerJump",
+    "ProgressivePowerSmash",
+    "ProgressiveQuakeHammer",
+    "ProgressiveSmashCharge",
+    "PulseStone",
+    "PyramidStone",
+    "QuakeBounce",
+    "QuakeHammer",
+    "QuickChange",
+    "RedBerry",
+    "RedKey",
+    "Refund",
+    "RepelGel",
+    "RightOn",
+    "RuinsKey",
+    "RunawayPay",
+    "SJumpCharge",
+    "SSmashChg",
+    "SecondDegreeCard",
+    "ShootingStar",
+    "ShrinkSmash",
+    "ShrinkStomp",
+    "ShroomCake",
+    "ShroomSteak",
+    "SilverCredit",
+    "SleepStomp",
+    "SleepySheep",
+    "SlowGo",
+    "SmashCharge",
+    "SnowmanBucket",
+    "SnowmanDoll",
+    "SnowmanScarf",
+    "Spaghetti",
+    "SpecialShake",
+    "SpeedySpin",
+    "SpicySoup",
+    "SpikeShield",
+    "SpinAttack",
+    "SpinSmash",
+    "StarBeam",
+    "StarPiece",
+    "StarStone",
+    "StinkyHerb",
+    "StoneCap",
+    "StopWatch",
+    "StoreroomKey",
+    "StrangeCake",
+    "StrangeLeaf",
+    "SuperFocus",
+    "SuperJump",
+    "SuperShroom",
+    "SuperSmash",
+    "SuperSoda",
+    "Sushie",
+    "SushieUpgrade",
+    "SweetShroom",
+    "TastyTonic",
+    "ThirdDegreeCard",
+    "ThreeStarPieces",
+    "ThunderBolt",
+    "ThunderRage",
+    "ToyTrain",
+    "TripleDip",
+    "TubbaCastleKey",
+    "UltraShroom",
+    "UltraStone",
+    "VolcanoVase",
+    "VoltShroom",
+    "WarehouseKey",
+    "WaterStone",
+    "Watt",
+    "WattUpgrade",
+    "WhackasBump",
+    "YellowBerry",
+    "YoshiCookie",
+    "YummyMeal",
+    "ZapTap"
+  ];
+  public filteredItems: string[] = this.PLANDO_ITEMS.slice();
+  public autocompleteSubscriptions: Subscription[] = [];
+  public searchText: FormControl;
+
+  ngOnInit(): void {
```

Comment:
I like having explicits public for every function, although I might have forgotten in some parts of the codebase.
Looks like initAutoComplete can be private though

Reply:
(Same comments applies to other components where the visibility on functions is missing)